content/concepts/transport.md

```diff
@@ -115,3 +115,98 @@ and some places in the codebase still use the "swarm" terminology.
 {{% /notice %}}
 
 [definition_switch]: /reference/glossary/#switch
+
+## QUIC
+
+QUIC is a new transport protocol that provides an always-encrypted, stream-multiplexed 
+connection built on top of UDP. It started as an experiment by Google on Google Chrome 
+in 2014, and was later standardized by the IETF in 
+[RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000).
+
+### Key challenges with TCP
+
+1. Head-of-line blocking (HoL blocking): TCP is a single byte stream exposed by the kernel, 
+   so streams layered on top of TCP experience HoL blocking.
+
+   {{% notice "info" %}}
+   HoL blocking occurs when a the head packet holds up the other packets in the transmission 
+   queue. 
+   {{% /notice %}}
+
+2. Ossification: Because TCP is unencrypted, middleboxes can inspect and modify
+   TCP header fields and may break unexpectedly when they encounter anything they don’t like.
+
+   {{% notice "info" %}}
+   Middleboxes are intermediary networking devices that can perform special functions to 
+   inspect, transform, and even manipulate network traffic. Examples of middleboxes are 
+   firewalls, NATs, proxies, and load balancers.
+   {{% /notice %}}
+
+3. Handshake inefficiency: the 3-way handshake is inefficient, as it spends 1-RTT on verifying 
+   the client’s address.
+
+   {{% notice "info" %}}
+   TCP requires a 3-way handshake to establish a reliable, bidirectional connection as it intends to.
+   {{% /notice %}}
+
+We need a transport protocol that:
+
+- Understands streams 
+- Overcomes HOL blocking (Head-of-line blocking)
+- Overcomes the latency of connection setup
+- Overcomes the ossification risks of TCP
+
+and, ideally, keeps the guarantees of TCP.
+
+A web browser connection typically entails the following (TCP+TLS+HTTP/2):
+
+1. Transport layer: TCP runs on top of the IP layer to provide a reliable 
+   byte stream.
+   - TCP provides a reliable, bidirectional connection between two end systems.
+2. Secure communication layer: A TLS handshake runs on top of TCP to,
+   establishing an encrypted and authenticated connection.
+   - Standard TLS over TCP requires 3-RTT. A typical TLS 1.3 handshake takes 1-RTT.
+3. Application layer: HTTP runs on a secure transport connection to transfer 
+   information and applies a stream muxer to serve multiple requests.
+   - Application data starts to flow.
+
+<!-- to add diagram -->
+
+### What is QUIC?
+
+QUIC combines the functionality of these layers: it sends UDP packets. Therefore, 
```

Comment:
```suggestion
QUIC combines the functionality of these layers. It sends UDP packets and therefore
```
This phrasing is imo a bit misleading. I would either remove the colons or make the following points a bullet-point list.